README.md

```diff
@@ -53,6 +49,19 @@ Read more on how to deploy Monocular [here](chart/monocular/README.md).
 - [Deployment](chart/monocular/README.md)
 - [Development](docs/development.md)
 
+## Looking for an in-cluster Application management UI?
+
+To focus on the CNCF Helm Hub requirements, in-cluster features have been
+removed from Monocular 1.0 and above. We believe that providing a good solution
+for deploying and managing apps in-cluster is an orthogonal user experience to a
+public search and discovery site. There is other tooling that can support this
```

Comment:
other tooling vs other tool?

Reply:
hmm any particular reason to use tools over tooling?

Reply:
Ah I see, no we also mention the RedHat Automation Broker